CLI Tools: --configuration switch does not work

When I add the `--configuration Debug` option to my `dotnet ef` commands, it does not work:

```
$ dotnet ef --configuration Debug database drop -f
/home/caleb/Projects/caleb/Pomelo.EntityFrameworkCore.MySql/test/Pomelo.EntityFrameworkCore.MySql.PerfTests/obj/Pomelo.EntityFrameworkCore.MySql.PerfTests.csproj.EntityFrameworkCore.targets(4,5): error MSB4006: There is a circular dependency in the target dependency graph involving target "GetEFProjectMetadata". [/home/caleb/Projects/caleb/Pomelo.EntityFrameworkCore.MySql/test/Pomelo.EntityFrameworkCore.MySql.PerfTests/Pomelo.EntityFrameworkCore.MySql.PerfTests.csproj]
Unable to retrieve project metadata. Ensure it's an MSBuild-based .NET Core project. If you're using custom BaseIntermediateOutputPath or MSBuildProjectExtensionsPath values, Use the --msbuildprojectextensionspath option.
```

Without the switch everything works fine:

```
$ dotnet ef database drop -f

Build succeeded.
    0 Warning(s)
    0 Error(s)

Time Elapsed 00:00:02.63
Dropping database 'mysqltest'.
Database 'mysqltest' did not exist, no action was taken.
```

Looks like other people have problems with this too on StackOverflow: http://stackoverflow.com/questions/43676017/dotnet-ef-run-with-configuration-causes-msb4006

### Further technical details
EF Core version:

```
    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="1.1.0" />
    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="1.0.0" />
```

Database Provider: `Pomelo.EntityFrameworkCore.MySql 1.1.1`
Operating system: `Ubuntu 16.04 x64`

--- a/src/dotnet-ef/Project.cs
+++ b/src/dotnet-ef/Project.cs
@@ -68,7 +68,7 @@ namespace Microsoft.EntityFrameworkCore.Tools
             try
             {
                 var propertyArg = "/property:EFProjectMetadataFile=" + metadataFile;
-                if (configuration != null)
+                if (framework != null)
                 {
                     propertyArg += ";TargetFramework=" + framework;
                 }

